docs/blog/rendered/ibis-version-6.1.0-release.ipynb

```diff
@@ -0,0 +1,1226 @@
+{
```

Comment:
<p>Perhaps this is now incorrect since it appears in memory tables <em>are</em> supported! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
not sure why this is showing `False`, reworded to avoid